src/TCommandLine.cpp

```diff
@@ -62,215 +62,148 @@ TCommandLine::TCommandLine( Host * pHost, TConsole * pConsole, QWidget * parent
 
     QString spell_aff = path + pHost->mSpellDic + ".aff";
     QString spell_dic = path + pHost->mSpellDic + ".dic";
-    mpHunspell = Hunspell_create( spell_aff.toLatin1().data(), spell_dic.toLatin1().data() );//"en_US.aff", "en_US.dic");
+    mpHunspell = Hunspell_create(spell_aff.toLatin1().data(), spell_dic.toLatin1().data()); //"en_US.aff", "en_US.dic");
     mpKeyUnit = mpHost->getKeyUnit();
     setAutoFillBackground(true);
     setFocusPolicy(Qt::StrongFocus);
 
     QFont font = mpHost->mDisplayFont;
     setFont(font);
 
-    mRegularPalette.setColor(QPalette::Text, mpHost->mCommandLineFgColor );//QColor(0,0,192));
-    mRegularPalette.setColor(QPalette::Highlight,QColor(0,0,192));
-    mRegularPalette.setColor(QPalette::HighlightedText, QColor(255,255,255));
-    mRegularPalette.setColor(QPalette::Base,mpHost->mCommandLineBgColor);//QColor(255,255,225));
+    mRegularPalette.setColor(QPalette::Text, mpHost->mCommandLineFgColor); //QColor(0,0,192));
+    mRegularPalette.setColor(QPalette::Highlight, QColor(0, 0, 192));
+    mRegularPalette.setColor(QPalette::HighlightedText, QColor(255, 255, 255));
```

Comment:
I suspect this is `Qt::white` isn't it - which is easier on a (human) parser... :grinning: